setup.py

```diff
@@ -91,7 +91,7 @@ def get_version(version_file):
         # Celery 5.0 introduces breaking changes to the CLI. See
         # https://docs.celeryq.dev/en/stable/history/whatsnew-5.0.html?highlight=command%20line%20options#new-command-line-interface
         # for more information.
```

Comment:
I don't think this comment is still necessary.

Reply:
Thanks for removing that comment in https://github.com/cisagov/admiral/pull/17/commits/32643280f57ea897cc504a1de949652123d8aa86.